public/index.html

```diff
@@ -14,6 +14,6 @@
   </div>
 
   <div id="root"></div>
-  <script src="./bundle.js"></script>
+  <script src="/./bundle.js"></script>
```

Comment:
Если js файл лежит рядом с `index.html` то можно писать без дополнительных символов.
```suggestion
  <script src="bundle.js"></script>
```
`/` означает что js файл лежит в корне структуры сайта
`./` означает ровно то же что и без начальных символов